platform/ui-next/src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -246,11 +255,11 @@ const Thumbnail = ({
         className,
         'bg-muted hover:bg-primary/30 group flex cursor-pointer select-none flex-col rounded outline-none',
         viewPreset === 'thumbnails' && 'h-[170px] w-[135px]',
-        viewPreset === 'list' && 'col-span-2 h-[40px] w-[275px]'
+        viewPreset === 'list' && 'h-[40px] w-full'
       )}
       id={`thumbnail-${displaySetInstanceUID}`}
       data-cy={
-        thumbnailType === 'thumbnailNoImage'
+        componentType === 'thumbnailNoImage'
```

Comment:
is there really a need for this rename? it will be a breaking change that we need to document